bridges/ethMultiversX/bridgeExecutor.go

```diff
@@ -169,6 +185,47 @@ func (executor *bridgeExecutor) StoreBatchFromMultiversX(batch *bridgeCore.Trans
 	return nil
 }
 
+// addBatchSCMetadataMvx fetches the logs containing sc calls metadata for the current batch
+func (executor *bridgeExecutor) addBatchSCMetadataMvx(ctx context.Context, transfers *bridgeCore.TransferBatch) (*bridgeCore.TransferBatch, error) {
+	if transfers == nil {
```

Comment:
I'd move this check on L162.. at this point, the code would have aready panicked while using batch.ID if this was nil